README.md

```diff
@@ -13,9 +13,11 @@ Loaded values are also injected into the local environ. This is to assist with a
 ---
 
 ### Requirements
+
```

Comment:
Curious, which formatter are you using on `md` files? I'm not against the changes here and would like to add it to my own tool set/pre-commit hooks.